examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/0_Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<blockquote>In this tutorial, we showed how to migrate from local Python functions to algorithms running on AWS.</blockquote><p>In this tutorial, we showed how to migrate from local Python functions to running algorithms on simulators and QPUs on Amazon Braket.</p><p> </p><blockquote>It was beneficial to run as a hybrid job so that we offload all classical compute onto AWS EC2, and retain priority queueing for our iterative algorithm.</blockquote><p>Using Amazon Braket Hybrid Jobs allowed us to run algorithms asynchronously, scale classical compute out into AWS, and obtain priority access to the selected QPU for the duration of our algorithm. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>